apps/nerves_hub_web_core/lib/nerves_hub_web_core/devices.ex

```diff
@@ -339,6 +339,20 @@ defmodule NervesHubWebCore.Devices do
     end
   end
 
+  @spec get_jitp_by_ski(binary) :: {:ok, CACertificate.JITP.t()} | {:error, any()}
+  def get_jitp_by_ski(ski) do
+    Repo.get_by(CACertificate, ski: Base.decode64!(ski))
```

Comment:
In case of invalid `ski`  field, shouldn't we return 400 or so ?

Reply:
good suggestion, thanks